docs/testing-code.md

```diff
@@ -88,7 +88,7 @@ $ npm -s run test:unit -- --browsers 'IE11 - Win7'
 ```
 
 !!! note
-    uData ensure compatibility for IE [officialy supported by Microsoft][ie-support].
+    uData ensure compatibility for IE [officially supported by Microsoft][ie-support].
```

Comment:
Do you mean: uData tests ensure compatibility with the version of IE officially supported by Microsoft, which is currently IE 11?